src/utils.js

```diff
@@ -77,10 +79,17 @@ export const getPositionOfCapturingGroup = (markup, parameterName) => {
   if (indexDisplay === null) indexDisplay = indexId
   if (indexId === null) indexId = indexDisplay
 
-  if (parameterName === 'id') return sortedIndices.indexOf(indexId)
-  if (parameterName === 'display') return sortedIndices.indexOf(indexDisplay)
-  if (parameterName === 'type')
-    return indexType === null ? null : sortedIndices.indexOf(indexType)
+
+  if(regex && countCapturingGroups(regex) === 0) {
+    // custom regex does not use any capturing groups, so use the full match for ID and display
+    return parameterName === "type" ? null : 0;
+  }
+
+  let argOffset = 1; // first argument always is the full match
```

Comment:
Please use `const` if the value doesn't change.

Reply:
Sure, I took the code that was previously in the custom-regex branch and changed `var`s to `let`s, but this one is definitely a `const` ^^

Reply:
👌